components/docs/DocsCards.tsx

```diff
@@ -0,0 +1,59 @@
+import Link from 'next/link';
+import React from 'react';
+
+import { HeadingLevel, HeadingTypeStyle } from '@/types/typography/Heading';
+import { ParagraphTypeStyle } from '@/types/typography/Paragraph';
+
+import { buckets } from '../data/buckets';
+import Heading from '../typography/Heading';
+import Paragraph from '../typography/Paragraph';
+
+type IconType = (props: { className: string }) => JSX.Element;
+
+interface CardProps {
+  title: string;
+  description: string;
+  link: string;
+  className: string;
+  Icon: IconType;
+}
+
+const Card: React.FC<CardProps> = ({ title, description, link, className, Icon }) => {
```

Comment:
Use `export default` to export the component right from here to follow the same coding style.

Reply:
This Card Component is not being used outside this file.